### DISCUSSION TITLE
Cannot reproduce Materials Project energy for ZnCu (mp-987)

### DISCUSSION THREAD

**Discussion by KonniJakob** [materialsproject/pymatgen#3982] [Q&A] [answered]
Hi there!

During calculations on a ZnCu structure from the Materials Project (mp-987), I noticed that I cannot reproduce the energies provided on their website, despite using parameters from MPRelaxSet in pymatgen. The energies obtained from VASP relaxations lie approx. 500 meV (i.e. ~250 meV/atom) above the reported entries in Materials Project.

For the calculations, I used the ZnCu structure directly downloaded from MP (see POSCAR). Then, I prepared the other input files using MPRelaxSet from pymatgen.io.vasp, see INCAR, KPOINTS, POTCAR, and create_vasp_input.py files attached. (This step has been tested with pymatgen versions 2022.9.8 and 2024.4.13 and yields identical files.) Finally, I obtain a VASP energy of -5.025 eV (i.e. -2.513 eV/atom), see OUTCAR or vasprun.xml attached, whereas MP reports -5.54 eV (i.e. -2.77 eV/atom). 

I have asked several colleagues to reproduce this result and all calculations end up at around the same values ranging from -5.00 eV to -5.05 eV, even with varying k-point grids, different initial magnetic moments, different VASP versions, or even different pseudopotentials. Furthermore, I confirmed that I used exactly the same POTCAR versions as in the original Materials Project relaxation (PAW_PBE Zn from 06Sep2000 & PAW_PBE Cu_pv from 06Sep2000). 

Could you please validate these observations or point me towards potential sources of error in this workflow?

Thanks in advance for your help and feel free to reach out, should you require more details!

Greetings,
Konstantin

[ZnCu_MP.tar.gz](https://github.com/user-attachments/files/16550883/ZnCu_MP.tar.gz)

**Comment by DanielYang59** [comment, marked as answer]
I tried to recreate the energy and ended up getting similar results to yours (-5.00 eV) using VASP 5.4.4, with exactly [the same INCAR tags listed on the database](https://next-gen.materialsproject.org/materials/mp-987/tasks/mp-987) (except for `NPAR = 4` for my particular cluster), not `MPRelaxSet`. For PSP of copper, I tried both `Cu` ([recommended by VASP](https://www.vasp.at/wiki/index.php/Available_pseudopotentials)) and `Cu_pv`([recommended by MP](https://docs.materialsproject.org/methodology/materials-methodology/calculation-details/gga+u-calculations/pseudopotentials)), and didn't noticed much difference. 

However, note that original calculation was carried out with VASP 5.2.2, and I'm not sure if this deviation is related to VASP version (I didn't manage to compile it with AMD AOCC toolchain on my EPYC server, I might give it another try later and keep you updated).

**Reply by KonniJakob** [reply]
Hi DanielYang59!

Thanks for looking into this topic and running some calculations to confirm our results. It turns out that there are significant and non-obvious changes in the respective POTCAR files of VASP 5.2.2 and VASP 5.4.4, as you suspected. In particular, both POTCAR files are tagged with the name '06Sep2000' despite the fact that they have been modified later on, leading to our confusion. 

Note also that these differences occur for a variety of different chemical systems beyond ZnCu (see figure attached), even though in our investigations the differences were most pronounced in that specific case.

Greetings,
Konstantin

![VASP_ref_diff_scatter_init_energies](https://github.com/user-attachments/assets/846fe8e4-08e0-4355-a1b3-b9c6f05c5e12)

**Reply by DanielYang59** [reply]
Thanks for sharing the additional findings, and great to know your sorted it out (Sorry I still haven't got much time to compile VASP 5.2 with AOCC). And thanks @esoteric-ephemera for the input as well.

**Comment by esoteric-ephemera** [comment]
@KonniJakob : VASP uses little to no version control for pseudopotentials, so many pseudopotentials have been updated without changing the name (TITEL)

More, you cannot compare absolute total energies from the same functional but different pseudopotentials. The pseudopotential vacuum level changes unpredictably. See [this paper](https://www.osti.gov/servlets/purl/1831768) for the theory behind this

You should still get roughly similar energy differences (formation, cohesive, etc.) using different pseudos than MP uses
